plutus-core/untyped-plutus-core/test/Evaluation/Golden/iteAtIntegerFullyApplied.type.golden

```diff
@@ -1,6 +1,6 @@
 (Left Type mismatch at () in term
   '(con string "11 <= 22")'.
 Expected type
-  '(con string)',
+  '[(con []) (con char)]',
```

Comment:
`String` is literally `[Char]` and due to `Note [Normalization of built-in types]` we have to normalize polymorphic built-in types by replacing intra-universe application with `TyApp`, which is how `(con string)` becomes `[(con []) (con char)]`.

Reply:
Right, but there are a number of golden tests which are supposed to fail but I think they're now failing for the wrong reason.  This one is instantiating `ifThenElse` at `integer` but returning a string, but now the failure's because it's got the wrong sort of string, I think.

Reply:
The diff is

```
- '(con string)',
+ '[(con []) (con char)]',
```

and the rest 

```
(Left Type mismatch at () in term
  '(con string "11 <= 22")'.
Expected type
  <...>,
found type
  '(con integer)')
```

is still the same. I.e. the expected type is now reported in normalized form (the new one) and that's it. Or am I missing something?

Reply:
(I think the diff is fine and Kenneth is vacationing, so I'm resolving this)